### PR TITLE
games by season method

### PR DESCRIPTION
**Pull request by exasperlnc**
### Describe your changes
- create games by seasons method to access and sort season/game data. 
- create game team class and respective testing
- 

### Checklist before requesting a review
- [ x] I have performed a self-review of my code.
- [x] I have added thorough tests - unit and integrated.

### Number of teammates to review this PR before merging
- [ x] one 
- [ ] two 
- [ ] EVERYONE PLS

### CODE REVIEW

File: spec/game_team_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    )
+  end
+
+  it 'exists' do
+    expect(@game_team).to be_a(GameTeam)
+  end
+
+  it 'has attributes' do
+    expect(@game_team.game_id).to eq(2012030221)
+    expect(@game_team.team_id).to eq('3')
```
Comment (grace-joh) [inline spec/game_team_spec.rb]: I think we have been converting team_id's to be integers

File: lib/game_team.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+              :team_id,
+              :home_or_away, 
+              :result, 
+              :head_coach, 
+              :goals, 
+              :shots, 
+              :tackles
+  def initialize(row)
+   @game_id = row[:game_id].to_i
+   @team_id = row[:team_id]
```
Comment (grace-joh) [inline lib/game_team.rb]: .to_i ?

**Review by grace-joh — CHANGES_REQUESTED** [review]
team id's need to be integers to align with the rest of our classes!

File: lib/game_team.rb (hunk 2 above)
Comment (exasperlnc) [inline lib/game_team.rb]:
```suggestion
   @team_id = row[:team_id].to_i
```

File: spec/game_team_spec.rb (hunk 1 above)
Comment (grace-joh) [reply spec/game_team_spec.rb]:
```suggestion
    expect(@game_team.team_id).to eq(3)
```

**Review by grace-joh — APPROVED** [review]
"it is now done" -Logan

**Pull request merged**